content/en/docs/guides/wait-for-ready.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Wait For Ready
+description: >-
+  Explains how and why to configure the wait for ready feature.
+---
+
+### Overview
+
+This is a feature which can be used on a stub which will cause the rpcs to wait
+(until an optional deadline is exceeded or) for the server to become available
+before sending the request.
+
+This is useful for batch workflows where there is no need to fail fast as it
+allows simplifying the implementation by hiding transient failures.
+
+When an RPC is created while the channel is not in a READY state, without
+waitForReady it will immediately return a failure; with waitForReady it will
+simply block until the connection becomes ready.
+
+For detailed semantics see:  [grpc/doc/wait-for-ready.md][grpc doc]
+
+### How to use Wait for Ready
+
+waitForReady is an attribute associated with a stub. When an RPC is created, the
+wait for ready attribute is automatically provided to the associated stream.
+
+Warning:  the RPC can still fail for other reasons than the server not being
```

Comment:
"reasons than the server" -> "reasons besides the server" ?

Reply:
done